package.json

```diff
@@ -34,7 +34,7 @@
     "chalk": "^2.4.2",
     "cross-env": "^5.2.0",
     "if-env": "^1.0.4",
-    "mocha": "^5.2.0",
```

Comment:
I dont think we should be updating this file

Reply:
Reverted the file back to master version.